zoo/settings.py

```diff
@@ -76,8 +76,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'college',
+        'USER': 'vinci',
+        'PASSWORD': 'Abhishek1706',
+        'HOST': 'localhost',
```

Comment:
These are your personal settings I guess. I have updated the settings.py, so you can use that.